docs/modules/ROOT/pages/managing-viridian-clusters.adoc

```diff
@@ -0,0 +1,87 @@
+== Managing Viridian Clusters using CLC
+:description: In this tutorial, you'll learn the basics of managing Viridian clusters using CLC. You'll see how to create, list, delete clusters and also how to import data to them and how to download their logs. Also you'll learn how to perform stop/resume operations on Viridian clusters using CLC.
+
+{description}
+
+== Before You Begin
+
+Be sure you logged in to your Viridian account. To be able to log in to Viridian use the following command:
+
+[source, bash]
+----
+clc viridian login
+----
+
+It is going to ask for `API Key` and `API Secret`. You can create them on link:https://viridian.hazelcast.com/settings/developer[Viridian Developer Settings Page]. If your login is successfully completed, you'll see the following:
+
+[source, bash]
+----
+Viridian token was fetched and saved.
+OK
+----
+
+Then, you are ready to go to the next steps.
+
+== Creating a Cluster on Viridian
+To create a new cluster on Viridian, you can use the following command:
+
+[source, bash]
+----
+clc viridian create-cluster --name foo --development
+OK
+----
+This creates a development cluster named `foo`.
+
+== Downloading the Logs of the Viridian Cluster
```

Comment:
I think we should talk about downloading logs later in the tutorial.

Could you move listing the cluster section here?